app/models/article.rb

```diff
@@ -1,2 +1,17 @@
 class Article < ApplicationRecord
+  def even?
+    Time.current.to_i.even?
+  end
```

Comment:
⚠️ **[undercover]** <Untested instance method> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Instance method `even?` is missing coverage for line 3 (node coverage: 0.0)
<!-- __reviewdog__:ChBkMmFhZjc2N2IwNTlhNDA0Egp1bmRlcmNvdmVy -->